diff --git a/node_modules/@1stg/lint-staged/base.js b/node_modules/@1stg/lint-staged/base.js
index 827bd15..51c8e5d 100644
--- a/node_modules/@1stg/lint-staged/base.js
+++ b/node_modules/@1stg/lint-staged/base.js
@@ -1,7 +1,7 @@
 const { isPkgAvailable, tryFile } = require('@pkgr/utils')
 
 const config = [
-  '*.{*sh,env,env.*,gql,html,json,properties,pug,rb,svelte,vue,toml,yaml,yml}',
+  '*.{*sh,env,env.*,gql,html,json,properties,pug,rb,vue,toml,yaml,yml}',
   '.*rc',
   'Dockerfile',
 ].reduce(
