diff --git a/node_modules/eslint-mdx/lib/parser.js b/node_modules/eslint-mdx/lib/parser.js
index f7a955b..f151256 100644
--- a/node_modules/eslint-mdx/lib/parser.js
+++ b/node_modules/eslint-mdx/lib/parser.js
@@ -18,7 +18,7 @@ exports.DEFAULT_PARSER_OPTIONS = {
     ecmaFeatures: {
         jsx: true,
     },
-    ecmaVersion: new Date().getUTCFullYear(),
+    ecmaVersion: 2021,
     sourceType: 'module',
     tokens: true,
     filePath: exports.PLACEHOLDER_FILE_PATH,
